avocado-setup.py

```diff
@@ -645,6 +668,7 @@ def parse_test_config(test_config_file, avocado_bin):
 
     args = parser.parse_args()
     env_check()
+    pip_install()
```

Comment:
I think it is better if this is called inside `env_check()`

Reply:
ya make sense